packages/replay-internal/test/utils/use-fake-timers.ts

```diff
@@ -1,14 +1,5 @@
-export function useFakeTimers(): void {
-  const _setInterval = setInterval;
-  const _clearInterval = clearInterval;
-  jest.useFakeTimers();
-
-  let interval: any;
-  beforeAll(function () {
-    interval = _setInterval(() => jest.advanceTimersByTime(20), 20);
-  });
+import { vi } from 'vitest';
 
-  afterAll(function () {
-    _clearInterval(interval);
-  });
```

Comment:
Not sure what this was used for but this was causing all sorts of trouble in the tests.